### PR TITLE
Fix syntax error in ZCL_AJSON in 7.02

### LINKED ISSUE #133: Syntax error in ZCL_AJSON in 7.02
There is a syntax error in the test classes of `ZCL_AJSON` if you import it in a 7.02 system.

The tests are using `CL_ABAP_UNIT=>ASSERT_TRUE( ... )` and `CL_ABAP_UNIT=>ASSERT_FALSE( ... )`. These methods are not available in release 7.02.

See also abapGit/abapGit#5772

### PR DESCRIPTION
**Pull request by schneidermic0**
Closes #133

### CODE REVIEW

File: src/zcl_ajson.clas.testclasses.abap [hunk 1]
```diff
@@ -3438,13 +3438,15 @@ class ltcl_integrated implementation.
 
     li_cut = zcl_ajson=>create_empty( ).
 
-    cl_abap_unit_assert=>assert_true( li_cut->is_empty( ) ).
+    cl_abap_unit_assert=>assert_equals( exp = abap_true
+                                        act = li_cut->is_empty( ) ).
```
Comment (sbcgua) [inline src/zcl_ajson.clas.testclasses.abap:3442]: Could you please indent it to the next line (2 space from cl_...) like the `li_cut->set` below

**Review by sbcgua — CHANGES_REQUESTED** [review]
Good point, thank you ! One small request, could you please restyle the indentation, and I'll merge.

File: src/zcl_ajson.clas.testclasses.abap (hunk 1 above)
Comment (sbcgua) [reply src/zcl_ajson.clas.testclasses.abap:3442]: And the second call as well

Comment (larshp) [reply src/zcl_ajson.clas.testclasses.abap:3442]: I've opened https://github.com/abaplint/abaplint/issues/2702

Comment (schneidermic0) [reply src/zcl_ajson.clas.testclasses.abap]: Sure. Done.

Comment (schneidermic0) [reply src/zcl_ajson.clas.testclasses.abap]: Sure. Done.

**Review by sbcgua — APPROVED** [review]

**Comment by sbcgua** [comment]
Thanks ! Nice fix, merging :)

**Pull request merged**